modules/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/image_selector/js/ImageSelector.js

```diff
@@ -273,6 +287,7 @@ const ImageSelector = ({
 				boundingBox: rootNode,
 				dragAndDropArea: rootNode,
 				fileFieldName: 'imageSelectorFileName',
+				multipleFiles: false,
```

Comment:
This is not necessary because it doesn't work. So I would prefer to remove it an not create confussion (if this is false why are we checking the list size?)